commit 1acd9af9ff224fdc6e8ecb038be39490c987c128
Author:     Yan Cheng <58191769+yanchengnv@users.noreply.github.com>
AuthorDate: Fri May 5 12:25:24 2023 -0400
Commit:     GitHub <noreply@github.com>
CommitDate: Fri May 5 12:25:24 2023 -0400

    Api parity (#1722)
    
    * api dev
    
    * added missing funcs to flare api
    
    * fixes based on comments
    
    * added unit test case for job_def
    
    * changed copyright date
---
 nvflare/private/fed/server/server_engine.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nvflare/private/fed/server/server_engine.py b/nvflare/private/fed/server/server_engine.py
index 27469891..744b8dc3 100644
--- a/nvflare/private/fed/server/server_engine.py
+++ b/nvflare/private/fed/server/server_engine.py
@@ -735,7 +735,7 @@ class ServerEngine(ServerEngineInternalSpec):
             replies = self._send_admin_requests(requests, 15)
         result = {}
         for r in replies:
-            site_name = self.get_client_name_from_token(r.client_token)
+            site_name = r.client_name
             if r.reply:
                 error_code = r.reply.get_header(MsgHeader.RETURN_CODE, ReturnCode.OK)
                 if error_code != ReturnCode.OK:
